.circleci/ds005_outputs.txt

```diff
@@ -21,11 +21,15 @@ smriprep/sub-01/anat/sub-01_from-T1w_to-fsnative_mode-image_xfm.txt
 smriprep/sub-01/anat/sub-01_hemi-L_inflated.surf.gii
 smriprep/sub-01/anat/sub-01_hemi-L_midthickness.surf.gii
 smriprep/sub-01/anat/sub-01_hemi-L_pial.surf.gii
+smriprep/sub-01/anat/sub-01_hemi-L_pial.surf.json
```

Comment:
The extension is `.surf.gii`.

```suggestion
smriprep/sub-01/anat/sub-01_hemi-L_pial.json
```

Reply:
this is going to require a change within DDS, since:
https://github.com/nipreps/niworkflows/blob/dbd2a972d51b171c3914ee08248bfaf5543a757c/niworkflows/interfaces/bids.py#L625-L630